Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Dockerfile to build an optimized version of the Compass server.
+#
+# To run the production image locally, create a .env.production file with the
+# appropriate env vars (a copy of .env.local works - see NOTE below).
+#
+# Command to build and run a Docker image:
+#  % docker build . -t compass && docker run -p=3000:3000 --name=compass --rm --env-file=.env.production compass
+#
+# NOTE: If you have the references to `localhost` in the .env file for other
+# services like postgres, replace `localhost` with the local IP address, e.g.
+# using `% ipconfig getifaddr en0`. This is needed as the docker image run in
```

Comment:
`host.docker.internal` also resolves to the host IP

Reply:
Updated documentation.